ansible_base/settings/dynamic_settings.py

```diff
@@ -4,14 +4,42 @@
 #     Add a new requirements/requirements_<section>.in /even if its an empty file/
 #
 
+try:
+    ANSIBLE_BASE_FEATURES  # noqa: F821
+except NameError:
+    ANSIBLE_BASE_FEATURES = {}
 
-if ANSIBLE_BASE_FEATURES.get('AUTHENTICATION', False):  # noqa: F821
+try:
+    INSTALLED_APPS  # noqa: F821
+except NameError:
+    INSTALLED_APPS = []
+
+
+if ANSIBLE_BASE_FEATURES.get('SWAGGER', False):  # noqa: F821
+    if 'drf_spectacular' not in INSTALLED_APPS:  # noqa: F821
+        INSTALLED_APPS.append('drf_spectacular')  # noqa: F821
+
+
+if ANSIBLE_BASE_FEATURES.get('FILTERING', False):  # noqa: F821
+    REST_FRAMEWORK.update(  # noqa: F821
+        {
+            'DEFAULT_FILTER_BACKENDS': (
+                'ansible_base.filters.rest_framework.type_filter_backend.TypeFilterBackend',
+                'ansible_base.filters.rest_framework.field_lookup_backend.FieldLookupBackend',
+                'rest_framework.filters.SearchFilter',
+                'ansible_base.filters.rest_framework.order_backend.OrderByBackend',
+            )
+        }
+    )
+
+
+if 'ansible_base.authentication' in INSTALLED_APPS:
```

Comment:
now I'm a little confused, how is this block different from `ansible_base/authentication/settings.py`

Reply:
It's not, I need to clean this up. I was trying to do an include from dynamic_settings.py to `ansible_base/authentication/settings.py` but this got ugly because the test code was choking on the include. I like having the setting.py individually per application but testing feels important. 

I also want to figure out how to deal with the "feature_enabled" lines above this. I like being able to just look through `INSTALLED_APPS` to seei f something is being used but the swagger and filtering are not apps at this point.

Reply:
Removed `ansible_base/authentication/settings.py`